CHANGELOG.rst

```diff
@@ -2,6 +2,85 @@
 SDK Changelog
 =============
 
+3.1.0 (2025-03-24)
+==================
+
+Feature release
+
+There are additional changes compared to the alpha versions. Continue reading for the full
+changelog, which includes the alpha versions release notes.
+
+.. warning:: SDK 3.1.0 requires the backend API version 3.1.0 or newer.
+
+New features
+------------
+
+* [#4917] Changed placement of form "next", "previous", "continue later" and "log out"
+  buttons.
+* [#445] Adopting typescript in the SDK project.
+    - Added typescript dependency and config files.
+    - Added @open-formulieren/formio-renderer 0.4.0 dependency.
+    - Replaced a number of form field components with components from the new
+      formio-renderer library.
+    - Replaced a number of components and functions to typescript variants.
+    - Added typescript definitions for Form and Submission data.
+    - Upgraded a number of dependencies.
+    - Upgraded to @open-formulieren/design-tokens 0.58.0.
+    - Upgraded to @utrecht/components 7.4.0.
+* [#5046] The form start button will be hidden if the maximum number of submissions is
+  reached.
+* [#5033] Improved error message when service is unavailable.
+* [#2177] The map component now supports placing a marker, line or polygon.
```

Comment:
```suggestion
* [#2177] You can now also draw lines and polygons on map components, in addition to point markers.
```

We'll have to make a decision about https://github.com/open-formulieren/open-forms/issues/5191 too because it seems a bit broken now